docs/TESTS.md

```diff
@@ -1,44 +1,110 @@
-### Creating the project in Xcode
+## Xcode Tutorial
 
-Please see <a href="https://developer.apple.com/library/tvos/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/02-quick_start.html#//apple_ref/doc/uid/TP40014132-CH2-SW1" target="_blank">Apple's guide on XCTest</a>
+This guide explains how to create and configure an Xcode project for Exercism.
 
-These steps will use the `bob` exercise as an example. If you are not working on `bob`, replace `bob` with your exercise name in the steps below.
+These instructions are written with the expectation that some readers will be very new to Xcode. Each step is described in excruciating detail. However, with a little experience performing these instructions should only take a minute.
 
-* Select iOS-->Application and then Single View Application and click Next.
+_As far as housekeeping chores go, you are closer to the finish line than it appears._
 
-* Enter `bob-swift`* as your Product Name, set the language to Swift, and check "Include Unit Tests", and then click Next.
+### Create A New Xcode Project
+![001-splash](https://cloud.githubusercontent.com/assets/5385825/20957611/c756745a-bc05-11e6-8631-e9e738ad464b.png)
 
-* Save the project in the same folder as the Exercism test file:
+Select macOS and Cocoa Application. (iOS Single View Application can also be used, but choosing macOS avoids the time needed to launch the iOS simulator each time a test is run.)
 
-![screen shot 2016-02-29 at 1 50 17 pm](https://cloud.githubusercontent.com/assets/1554169/13405215/67123310-deeb-11e5-9a2e-7c375ee5fad6.png)
+![002-templatechooser](https://cloud.githubusercontent.com/assets/5385825/20957613/c770da16-bc05-11e6-82e1-7da6c2ca6de0.png)
 
-* You will see two targets in your project, one for the application named `bob-swift` and one for tests named `bob-swiftTests`. If you chose to create an OSX application instead of an iOS application, you will not see a `bob-swiftTests` target and will have to create it manually.
+#### Name The Project
+![003-nameproject](https://cloud.githubusercontent.com/assets/5385825/20957616/c7793828-bc05-11e6-8c73-bb02a71322b2.png)
 
-* Drag the Exercism test file into the bob-swiftTests group, ensure that "Copy items if needed" is checked, and add the line `@testable import bob_swift ` on the second line of the Exercism test file:
+1. Enter `Gigasecond` as the Product Name. This will become the name of the project. Note that any name can be used, but its helpful to avoid using punctuation such as hyphens, underscores, spaces, etc. _This walkthrough uses the `Gigasecond` exercise. For other exercises, replace the string `Gigasecond` with the exercise name._
```

Comment:
I agree that Hello World is a far better base for a tutorial, and had thought that eventually I'd use that in a future revision. 

The reason I didn't choose it now was twofold.

I followed the convention of the existing document, which also didn't use Hello world.

Second, when I first joined exercism a few months ago, the Swift track Hello World exercise was not especially helpful. (This was true at the time, it may no longer be true).

Since this was my first foray into contributing, I wanted to stick to what I knew as much as possible. That meant not touching Hello World yet. Once I learn more, I'd love a chance to refine this tutorial.